src/main/java/cafeboard/comment/CommentRepository.java

```diff
@@ -0,0 +1,9 @@
+package cafeboard.comment;
+
+import org.springframework.data.jpa.repository.JpaRepository;
+
+import java.util.List;
+
+public interface CommentRepository extends JpaRepository<Comment, Long> {
+    List<Comment> findByPostId(Long postId);
```

Comment:
별 건 아닌데 아래처럼 이름에 All을 끼워넣어도 에러 안 나긴 해요.
```suggestion
    List<Comment> findAllByPostId(Long postId);
```